dependency-injection/2.0/_index.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: "Jakarta Dependency Injection 2.0 (under development)"
-date: 2020-01-15
+title: "Jakarta Dependency Injection 2.0 (milestone)"
+date: 2020-05-18
 summary: "Release for Jakarta EE 9"
 ---
 
 Jakarta Dependency Injection specifies a means for obtaining objects in such a way as to maximize reusability, testability and maintainability compared to traditional approaches such as constructors, factories, and service locators (e.g., JNDI).
 
 * [Jakarta Dependency Injection 2.0 Release Record]()
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Dependency Injection 2.0 Specification Document]() (PDF)
-* [Jakarta Dependency Injection 2.0 Specification Document]() (HTML)
+* [Jakarta Dependency Injection 2.0 Specification Document](./injection-spec-2.0.pdf) (PDF)
```

Comment:
The spec file name should have some indication that it's not final and it's a Milestone or RC release.  Something like injection-spec-2.0-M1.pdf or injection-spec-2.0-RC4.pdf.  Otherwise, it could be confused with the final 2.0 release of the Spec.

Reply:
I have updated it to injection-spec-2.0-draft.* since internally the text indicates it is a draft.